README.md

```diff
@@ -6,27 +6,57 @@ BigQuery ETL
 Bigquery UDFs and SQL queries for building derived datasets.
 
 Recommended practices
-===
+---
+
+### Queries
+
+- Should be defined in files named as `sql/table_version.sql` e.g.
+  `sql/clients_daily_v6.sql`
+- Should not specify a project or dataset in table names to simplify testing
+- Should be [incremental](#incremental-queries)
+- Should filter input tables on partition and clustering columns
+- Should use `_` prefix in generated column names not meant for output
+- Should not use jinja templating on the query file in Airflow
+
+### UDFs
 
-- Queries
-  - Should be defined in files named as `sql/table_version.sql` e.g.
-    `sql/clients_daily_v6.sql`
-  - Should not specify a project or dataset in table names to simplify testing
-  - Should be [incremental](#incremental-queries)
-  - Should filter input tables on partition and clustering columns
-  - Should use `_` prefix in generated column names not meant for output
-  - Should not use jinja templating on the query file in Airflow
-- UDFs
-  - Should be used to avoid code duplication
-  - Should use lower snake case names with `udf_` prefix e.g. `udf_mode_last`
-  - Should be defined in files named as `udfs/function.{sql,js}` e.g. `udfs/udf_mode_last.sql`
-  - Should use `SQL` over `js` for performance
-  - Must not be used for incremental queries with a _mostly materialized view_ (defined below)
+- Should be used to avoid code duplication
+- Should use lower snake case names with `udf_` prefix e.g. `udf_mode_last`
+- Should be defined in files named as `udfs/function.{sql,js}` e.g. `udfs/udf_mode_last.sql`
+- Should use `SQL` over `js` for performance
+- Must not be used for incremental queries with a _mostly materialized view_ (defined below)
+
+### Backfills
+
+- Should be avoided on large datasets
+  - Backfills may double storage cost for a dataset for 90 days by moving
+    data from long-term storage to short-term storage
+    - For example regenerating `clients_last_seen_v1` from scratch would cost
+      about $1600 for the query and about $6800 for data moved to short-term
+      storage
+  - Should combine multiple backfills happening around the same time
```

Comment:
Maybe we should have a process for accumulating pending changes that would be convenient to include in the next backfill. Maybe one would PR non-urgent changes to a `.pending.sql` file.

Not in scope here, but something that popped into my head while reading.

Reply:
I was thinking that we could update queries to generate new columns and exclude old columns, and use views to hide changes, and then whenever we decide to backfill we would only have to run it up to the most recent change.